src/components/appContext/index.js

```diff
@@ -27,7 +27,7 @@ function getInitialFilterState() {
 
 const initialState = {
   filterOpen: null,
-  filteredEventsCount: 0,
+  eventsToShow: 9,
```

Comment:
are you pulling all of the data automatically from graphql or are you using pagination to pull 9 at a time?